### PR TITLE
V1.11.x merge

### PR DESCRIPTION
**Pull request by murgatroid99**

### CODE REVIEW

File: packages/grpc-js-core/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Pure JavaScript gRPC Client
 
+**Note: This is an alpha-level release. Some APIs may not yet be present and there may be bugs. Please report any that you encounter**
```
Comment (nicolasnoble) [inline packages/grpc-js-core/README.md]: That's an interesting change...

File: packages/grpc-js-core/src/call-stream.ts [hunk 2]
```diff
@@ -9,9 +9,10 @@ import {FilterStackFactory} from './filter-stack';
 import {Metadata} from './metadata';
 import {ObjectDuplex} from './object-stream';
 
-const {HTTP2_HEADER_STATUS, HTTP2_HEADER_CONTENT_TYPE, NGHTTP2_CANCEL} = http2.constants;
+const {HTTP2_HEADER_STATUS, HTTP2_HEADER_CONTENT_TYPE, NGHTTP2_CANCEL} =
+    http2.constants;
 
-export type Deadline = Date | number;
+export type Deadline = Date|number;
```
Comment (nicolasnoble) [inline packages/grpc-js-core/src/call-stream.ts]: We seem to have troubles deciding if we want spaces or not around these :D We really need a linter...

File: packages/grpc-js-core/README.md (hunk 1 above)
Comment (murgatroid99) [reply packages/grpc-js-core/README.md]: I added that on v1.11.x before we published. It's just getting into master here.

File: packages/grpc-js-core/src/call-stream.ts (hunk 2 above)
Comment (murgatroid99) [reply packages/grpc-js-core/src/call-stream.ts]: It's possible that this is a change by the formatter. Or me accidentally reversing a change by the formatter.

**Pull request merged**